pages/serializers.py

```diff
@@ -378,8 +378,30 @@ class Meta(EnumeratedElementSerializer.Meta):
             'certificate', 'viewing_duration',)
 
 
-class AttendanceInputSerializer(serializers.Serializer):
-    """
-    Serializer to validate input to mark users' attendance
-    to a LiveEvent(LiveEventAttendanceAPIView)
-    """
+class LiveEventSerializer(serializers.ModelSerializer):
+    scheduled_at = serializers.DateTimeField(
+        help_text=_('Date/time the live event is scheduled'))
+    rank = serializers.IntegerField(
+        help_text=_('Unique integer to denote the index of the LiveEvent'))
+    location = serializers.URLField(
+        help_text=_('URL to the live event'))
+    max_attendees = serializers.IntegerField(default=0,
+        help_text=_('Max attendees for the LiveEvent'))
+    status = serializers.CharField(read_only=True)
+
+    class Meta:
+        model = LiveEvent
+        fields = ('scheduled_at', 'rank', 'location', 'max_attendees', 'status')
+        read_only_fields = ('status',)
+
+
+class LiveEventAttendeesSerializer(EnumeratedProgressSerializer):
+    user = serializers.SerializerMethodField(
+        help_text=_("Username of the attendee"))
+
+    class Meta(EnumeratedProgressSerializer.Meta):
+        fields = ('user', 'content', 'viewing_duration', 'min_viewing_duration')
+        read_only_fields = EnumeratedProgressSerializer.Meta.read_only_fields + ('user',)
+
+    def get_user(self, obj):
```

Comment:
We might want to return a serialized User here at some point so it is easier for the UI to display the full name of the attendee. It has not yet been done in other parts of the API (ex: `Comment` serializer) so this most likely should be note/issue for a future refactoring. Good to keep consistency with existing APIs for now.